doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+
+
+If any of these elements do not apply to your project, include them as empty sections to prevent parsing errors.
+
+Example `pom-migration.json` file:
+
+[source,json,subs="attributes+"]
+----
+{
+  "dependencies":[
+
+    {"groupId":"junit", "artifactId":"junit", "version":"4.12", "scope":"test"},
+    {"groupId":"com.thoughtworks.xstream", "artifactId":"xstream", "version":"4.12", "scope":"test"}
+
+  ],
+
+  "repositories-add":[
+    {
+      "id":"jboss-public-repository-group",
+      "name":"JBoss Public Repository Group",
+      "url":"http://repository.jboss.org/nexus/content/groups/public/",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":true,
+      "snapshotUpdatePolicy":"never"
+    },
+    {
+      "id":"kie-internal-staging-repository-group",
+      "name":"KIE Internal Staging Repositories",
+      "url":"https://origin-repository.jboss.org/nexus/content/groups/kie-internal-group",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":false,
+      "snapshotUpdatePolicy":"never"
+    }
+  ],
+
+  "repositories-remove":[
+    {
+      "id":"productization-repository", "url":"http://download.lab.bos.redhat.com/brewroot/repos/jb-ip-6.1-build/latest/maven/"
+    }
+  ],
+
+  "repositories-update-urls":[
+    {
+      "id":"guvnor-m2-repo", "url":"http://localhost:8080/{URL_COMPONENT_CENTRAL}/maven3/"
+    }
+  ],
+
+  "pluginRepositories-add":[
+
+    {
+      "id":"jboss-public-repository-group",
+      "name":"JBoss Public Repository Group",
+      "url":"http://repository.jboss.org/nexus/content/groups/public/",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":true,
+      "snapshotUpdatePolicy":"never"
+    },
+    {
+      "id":"kie-internal-staging-repository-group",
+      "name":"KIE Internal Staging Repositories",
+      "url":"https://origin-repository.jboss.org/nexus/content/groups/kie-internal-group",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":false,
+      "snapshotUpdatePolicy":"never"
+    }
+
+  ],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+
+Example `pom-migration.json` file with `dependencies` only:
+
+[source,json]
+----
+{
+  "dependencies":[
+
+    {"groupId":"junit", "artifactId":"junit", "version":"4.12", "scope":"test"},
+    {"groupId":"com.thoughtworks.xstream", "artifactId":"xstream", "version":"4.12", "scope":"test"}
+
+  ],
+  "repositories-add":[],
+  "repositories-remove":[],
+  "repositories-update-urls":[],
+  "pluginRepositories-add":[],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+--
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
```

Comment:
We have this URL in the artifacts somewhere? Not sure, perhaps check with Stetson

Reply:
Not that I am aware of. This page is maintained by Martin Weiler in Support.